interfaces/cython/cantera/reactor.pyx

```diff
@@ -315,6 +315,31 @@ cdef class Reactor(ReactorBase):
         self.reactor.getState(&y[0])
         return y
 
+    property advance_limits:
+        """
+        Get or set absolute limits for state changes during `ReactorNet.advance`
+        (positive values are considered; negative values are ignored). 
```

Comment:
Should we raise or throw (since I guess it would be from C++) if negative values are passed? Hypothetically, negative values could be useful, e.g., fuel mole fraction decrease

Reply:
Limits are currently defined as absolute values (and thus apply to both increases and decreases). I thus was only checking for strictly positive values and set the default limit to -1 (no limit). I can accomplish the same behavior setting the default to 0 though.

Reply:
OK, I think absolute values make sense. I do think the user should be informed somehow that negative values are ignored, especially if negative values will turn off the limit for a particular component. How does a user turn off the limiting for a component if they want to?

I think `-1` is a fine sentinel value for C++, but I think it might be nice to have a way to disable a particular limit more naturally in Python, perhaps by giving it the value `False`, `None`, and/or `0`?

Reply:
Specifying `None` is certainly intuitive. Implementation (and test) will follow shortly.